.github/workflows/pr-tests-code.yml

```
@@ -0,0 +1,60 @@
+name: Run code tests
+
+on:
+  pull_request:
+    branches: 
+      - master
+      - develop
+    paths: 
+      - src/**
+
+jobs:
+  ubuntu:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v2
+      - name: Use .NET Core 2.1 LTS SDK
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: '2.1.x'
+      - name: Use .NET Core 3.1 LTS SDK
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: '3.1.x'
+      - name: Install dependencies
+        working-directory: src
+        run: dotnet restore
+      - name: Build solution (release config)
+        working-directory: src
+        run: dotnet build --no-restore -c Release -p:NoWarn=CS1591
+      - name: Build solution (debug config)
+        working-directory: src
+        run: dotnet build --no-restore -p:NoWarn=CS1591
```

```suggestion
        run: dotnet build --no-restore -c Debug -p:NoWarn=CS1591
```

Let's point that explicitly.